app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -110,24 +115,41 @@ function initializeCommentsPage() {
         function successCb(response) {
           var reactionCount = thisButt.querySelector('.reactions-count');
           var reactionLabel = thisButt.querySelector('.reactions-label');
+          var heartFilledSvg = thisButt.querySelector('.reaction-icon.reacted');
+          var heartNotFilledSvg = thisButt.querySelector('.reaction-icon.not-reacted');
           if (response.result === 'create') {
             thisButt.classList.add('reacted');
+            thisButt.ariaPressed = true;
             if (reactionCount) {
               reactionCount.innerHTML = parseInt(reactionCount.innerHTML) + 1;
               reactionCount.classList.remove("hidden");
               if(parseInt(reactionCount.innerHTML) == 1) {
                 reactionLabel.innerHTML = "&nbsp;like"
+                heartFilledSvg.querySelector('title').textContent = "Like Comment: 1 like"
```

Comment:
I think in general we should save `heartFilledSvg.querySelector('title')` to a variable, so it's not repeated in each branch

Reply:
Makes sense, Will do this change. @rhymes